region/brinstar/red/Red Brinstar Fireflea Room.json

```diff
@@ -499,7 +506,8 @@
           "mustStayPut": false
         }},
         {"refill": ["Energy", "PowerBomb"]}
-      ]
+      ],
+      "resetsObstacles": ["A"]
```

Comment:
good catch. I wonder if we might want to make `resetRoom` implicitly reset all obstacles?

Reply:
This could be awkward to make work properly in the randomizer. How about an alternative where we add a test to check that any strat with `resetRoom` has `resetsObstacles` covering all the obstacles? Either way, at the moment there's not a way to make the `resetsObstacles` conditional, so if `resetRoom` appears in an `or` then we could need to split those into separate strats, which will take some manual work.